NGitLab.Mock/Clients/MergeRequestClient.cs

```diff
@@ -571,6 +577,24 @@ public Models.MergeRequest Update(int mergeRequestIid, MergeRequestUpdate mergeR
                     }
                 }
 
+                if (mergeRequestUpdate.ReviewerIds != null)
+                {
+                    var users = new List<UserRef>();
+
+                    foreach (var userId in mergeRequestUpdate.ReviewerIds)
+                    {
+                        users.Add(Server.Users.GetById(userId));
+                    }
+
+                    if (!users.Any())
```

Comment:
put the check up to check on Server.Users.GetById(userId), update the error message with the not found user if that's the case.

No need to test on empty list